DependencyInjection/DoctrineExtension.php

```diff
@@ -407,6 +381,7 @@ protected function ormLoad(array $config, ContainerBuilder $container)
          * This is replaced with a true locator by ServiceRepositoryCompilerPass.
          * This makes that pass technically optional (good for tests).
          */
+        // @todo cleanup
```

Comment:
todo before merging?

Reply:
Ah, almost forgot about that. I'll update this 👍

Reply:
Done!